src/rpm/headers/header.rs

```diff
@@ -521,8 +532,8 @@ pub(crate) struct IndexHeader {
     pub(crate) version: u8,
     /// number of header entries
     pub(crate) num_entries: u32,
-    /// total header size excluding the fixed part (@todo: verify this is correct)
-    pub(crate) header_size: u32,
+    /// total amount of data stored
+    pub(crate) data_section_size: u32,
```

Comment:
I find `data` a very unforunate term, it leaves a lot of room for interpretation.

Reply:
It's how it's described by the documentation https://rpm-software-management.github.io/rpm/manual/format.html

The previous name was straight up incorrect in any case